src/tutorial/pet-shop/index.md

```diff
@@ -0,0 +1,773 @@
+---
```

Comment:
Can you explain how the redirect works?
Why do we need to duplicate the tutorial three times now?

Reply:
`mkdocs redirect` works only pointing an old rote to a new one (1-1). In our case, we have the aggravating factor that every directory is a route (`use_directory_urls = true`), so, we need to physically create all the routes and then put these routes in the routing configuration. In the given case, the petshop was inside the tutorial folder. So we were able to redirect only the tutorial/index.md route to guides/pet-shop/index.md

As new routes appeared (tutorial, tutorials...) we need to implement them physically (create folders and files) and after that, put the 1 to 1 in the yml configuration file.

  - redirects:
      redirect_maps:
        'tutorial/index.md': 'guides/pet-shop/index.md'
        'tutorial/pet-shop/index.md': 'guides/pet-shop/index.md'          
        'tutorials/index.md': 'guides/pet-shop/index.md'
        'tutorials/pet-shop/index.md': 'guides/pet-shop/index.md' 

Unfortunately this is the only way to do the redirect using this plugin. More information here: https://github.com/mkdocs/mkdocs-redirects

Maybe @kevinbluer or @cds-amal might have an idea how to do it another way